message/messaging_service.hh

```diff
@@ -409,6 +412,11 @@ public:
     future<> unregister_gossip_digest_ack2();
     future<> send_gossip_digest_ack2(msg_addr id, gms::gossip_digest_ack2 msg);
 
+    // Wrapper for GOSSIP_GET_ENDPOINT_STATES
+    void register_gossip_get_endpoint_states(std::function<future<gms::gossip_get_endpoint_states_response> (const rpc::client_info& cinfo, gms::gossip_get_endpoint_states_request request)>&& func);
+    future<> unregister_gossip_get_endpoint_states();
+    future<gms::gossip_get_endpoint_states_response> send_gossip_get_endpoint_states(msg_addr id, gms::gossip_get_endpoint_states_request request);
+
     // Wrapper for DEFINITIONS_UPDATE
     void register_definitions_update(std::function<rpc::no_wait_type (const rpc::client_info& cinfo, std::vector<frozen_mutation> fm,
```

Comment:
Do we not need a new cluster feature to know we may send the new verb?

In general I prefer to avoid adding new verbs if not strictly needed, but I think this one is such a simplification it is worth it.

Reply:
We can not use gossip features. This stuff is even before gossip starts

Reply:
But, we can remember if we've seen the feature before or not.

OTOH, we'll only have the bad verb during upgrades to this version. Maybe we can tolerate it.